content/influxdb/clustered/install/deploy.md

```diff
@@ -9,6 +9,46 @@ menu:
 weight: 104
 ---
 
+Use Kubernetes and related tools to deploy your InfluxDB cluster.
+This guide provides instructions for deploying your InfluxDB cluster using the
+following tools:
+
+- **kubectl**: CLI for controlling the Kubernetes cluster manager
+- **kubit**: A Kubernetes controller that can render and apply jsonnet
+  templates based on the [kubecfg](https://github.com/kubecfg/kubecfg) jsonnet
+  tooling and framework
+
+InfluxDB Clustered uses an `AppInstance` Kubernetes custom resource (CR) to
+configure and deploy your InfluxDB Cluster.
+Installing a CR requires cluster-wide permissions and may cause `kubectl` to
```

Comment:
```suggestion
Installing a `CustomResourceDefinition` (CRD) requires cluster-wide permissions and may cause `kubectl` to
```

My apologies here, my explanation was a little poor on the CR vs CRD:

- `CustomResourceDefinition` is the API object in Kubernetes which defines the type.
- Custom Resource is an actual object which is not native to Kubernetes, watched by a custom controller. For example, an `AppInstance` is a custom resource watched by `kubit`.

It is the CRD itself which requires elevated permissions to install, because it installs a new object onto the entire cluster, i.e. that object becomes available to use in every namespace.